packages/neuron-ui/src/components/NervosDAODetail/CellsCard/index.tsx

```diff
@@ -71,12 +73,36 @@ const TabsVariantWithCellsCard: FC<
                   t('transaction.cell-from-cellbase')
                 ) : (
                   <>
-                    <span className={styles.address}>{`${address.slice(0, 20)}...${address.slice(-20)}`} </span>
-                    <ScriptTag isMainnet={isMainnet} script={cell.lock} onClick={() => setShowingLockInfo(cell.lock)} />
+                    <Tooltip
+                      tip={
+                        <CopyZone content={address} className={styles.copyTableAddress}>
+                          {address}
+                        </CopyZone>
+                      }
+                      className={styles.addressTips}
+                      showTriangle
+                      isTriggerNextToChild
+                    >
+                      <span className={styles.address}>{`${address.slice(0, 20)}...${address.slice(-20)}`} </span>
```

Comment:
I recommend taking a peek at the [react-middle-ellipsis](https://github.com/bluepeter/react-middle-ellipsis) if we should truncate something in the middle in the future

Reply:
For the safety wallet, we will use fewer third-party libs because they may be out of control.

Reply:
> For the safety wallet, we will use fewer third-party libs because they may be out of control.

exactly, I forget that Neuron is a crypto wallet